### PR TITLE
CLDC-3028 Allow bulk uploading for merged organisations

### PR DESCRIPTION
**Pull request by kosiakkatrina**

### CODE REVIEW

File: app/models/organisation.rb [hunk 1]
```diff
@@ -25,6 +25,11 @@ def affiliated_stock_owners
       ids << id
     end
 
+    absorbed_organisations.each do |organisation|
+      ids.concat(organisation.stock_owners.pluck(:id))
```
Comment (natdeanlewissoftwire) [inline app/models/organisation.rb:29]: aren't these stock owner relationships applied to the absorbing organisation during a merge?

File: app/models/organisation.rb [hunk 2]
```diff
@@ -25,6 +25,11 @@ def affiliated_stock_owners
...
+      ids << organisation.id if organisation.holds_own_stock?
```
Comment (natdeanlewissoftwire) [inline app/models/organisation.rb:30]: isn't this condition always true?

File: app/models/organisation.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/models/organisation.rb:29]: Hmm, I guess if they're removed on absorbed orgs but they might still be relevant for merged 🤔

File: app/models/organisation.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/models/organisation.rb:30]: What makes you think that? I think merged organisation might not hold stock (?) Am I missing something?

File: app/models/organisation.rb (hunk 1 above)
Comment (natdeanlewissoftwire) [reply app/models/organisation.rb:29]: I'm not sure what you mean - I'm saying that this line is already covered by line 33 if that helps

File: app/models/organisation.rb (hunk 2 above)
Comment (natdeanlewissoftwire) [reply app/models/organisation.rb:30]: Ah no you're right, I was still thinking we were looking at stock owners of this org, ignore me

**Review by natdeanlewissoftwire — APPROVED** [review]

**Pull request merged**